content/api/rest-sdk/channels.textile

```diff
@@ -0,0 +1,288 @@
+---
+title: Channels
+meta_description: "Client Library SDK REST API Reference Channels documentation."
+meta_keywords: "Ably, Ably REST, API Reference, REST SDK, REST interface, REST API, Channels"
+section: api
+index: 20
+languages:
+  - javascript
+  - nodejs
+  - php
+  - python
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - go
+  - flutter
+jump_to:
+  Channels API methods:
+    - get
+  Channel API properties:
+    - name
+    - presence
+  Channel API methods:
+    - publish
+    - history
+  Types:
+    - Related Types#related-types
+redirect_from:
```

Comment:
I think this should be removed otherwise there will be 2 pages redirecting from that URL.